### PR TITLE
docs: add release instructions

### PR DESCRIPTION
**Pull request by rjfd**
Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: HOWTORELEASE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Release Procedure
+
+These are the steps to make a release for version `<version_number>`:
+
+1. Create a commit with title `Bump to v<version_number>` with the following changes:
+    1. Update `CHANGELOG.md` with all important changes introduced since previous version.
```
Comment (smithfarm) [inline HOWTORELEASE.md:6]: in outlines, it's considered suboptimal to have `1.` and `2.` numbering under `1.`. Can we change the sublevel numbering to `a.`, `b.` ?

Comment (smithfarm) [reply HOWTORELEASE.md:6]: The modifications of file `CHANGELOG.md` have to adhere to some kind of syntax, right? Would it make sense to mention that and add a link to the changelog syntax definition?

Comment (rjfd) [reply HOWTORELEASE.md:6]: yes we can!

File: HOWTORELEASE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        - Create a new section `[<version_number>] <date YYYY-MM-DD>` and move all entries
+          from the `[Unreleased]` section to the new section.
+        - Make sure all github issues resolved in this release are referenced in the changelog.
+        - Update the links on the bottom of the file.
```
Comment (smithfarm) [inline HOWTORELEASE.md]: s/on the bottom/at the bottom/

File: HOWTORELEASE.md (hunk 1 above)
Comment (rjfd) [reply HOWTORELEASE.md:6]: done

File: HOWTORELEASE.md (hunk 2 above)
Comment (rjfd) [reply HOWTORELEASE.md]: done

File: HOWTORELEASE.md (hunk 1 above)
Comment (rjfd) [reply HOWTORELEASE.md:6]: The changelog file has the link to syntax and structure

File: HOWTORELEASE.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+These are the steps to make a release for version `<version_number>`:
+
+1. Create a commit with title `Bump to v<version_number>` with the following changes:
+    a. Update `CHANGELOG.md` with all important changes introduced since previous version.
+        - Create a new section `[<version_number>] <date YYYY-MM-DD>` and move all entries
+          from the `[Unreleased]` section to the new section.
+        - Make sure all github issues resolved in this release are referenced in the changelog.
+        - Update the links at the bottom of the file.
+    b. Update version number in `ceph-bootstrap.spec` to `Version: <version_number>`.
+2. Create an annotated tag for the above commit with `git tag -s -a v<version_number>`.
```
Comment (smithfarm) [inline HOWTORELEASE.md]: Create an annotated tag for the above commit: 

(drop "with", replaced with `:`)

File: HOWTORELEASE.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+1. Create a commit with title `Bump to v<version_number>` with the following changes:
+    a) Update `CHANGELOG.md` with all important changes introduced since previous version.
+        - Create a new section `[<version_number>] <date YYYY-MM-DD>` and move all entries
+          from the `[Unreleased]` section to the new section.
+        - Make sure all github issues resolved in this release are referenced in the changelog.
+        - Update the links at the bottom of the file.
+    b) Update version number in `ceph-bootstrap.spec` to `Version: <version_number>`.
+2. Create an annotated tag for the above commit with `git tag -s -a v<version_number>`.
+    - The message should be `version <version_number>`.
```
Comment (smithfarm) [inline HOWTORELEASE.md:13]: - Using `git show`, review the commit message of the annotated tag. It should say: `version <version_number>`.

**Review by smithfarm — APPROVED** [review]
subject to nits

**Pull request merged**